src/richie/apps/core/templatetags/language_name.py

```diff
@@ -0,0 +1,23 @@
+"""Custom template tags related to FEATURES settings."""
+from django import template
+from django.conf import global_settings
+from django.utils import translation
+
+# do not use the lazy
+# pylint: disable=import-outside-toplevel
+from django.utils.translation import gettext as _
+
+register = template.Library()
+
+
+@register.simple_tag()
+def get_original_language_name(language_code):
+    """Get language original name on that its locale"""
+    # Convert language code from 'fr-ca' to 'fr'
+    code = language_code.split("-")[0]
```

Comment:
Shouldn't we try to retrieve first the translation through the full code ?
e.g: We could have a language selector which display 
- `French` for `fr-FR`
- `French Canadian` for `fr-CA`

Reply:
Yes. After your comment, I replaced the use of `globalsettings` to the normal `settings`.
Because we have the same `LANGUAGES` but only the ones that are configured in our installation.